__tests__/dockerfile.test.ts

```diff
@@ -1,26 +1,16 @@
 import {test, expect} from '@jest/globals'
 import * as path from 'path'
 import {load} from '../src/dockerfile'
-import {AlpineImage, DebImage} from '../src/image'
 
-test('load invalid dockerfile', () => {
-  let dockerfilePath = path.join(__dirname, 'data', 'InvalidDockerfile')
-  function loadInvalid() {
-    load(dockerfilePath)
-  }
-  expect(loadInvalid).toThrowError('Unable to extract image from Dockerfile')
-})
-
-test('load alpine dockerfile', () => {
+test('load alpine dockerfile', async () => {
   const dockerfilePath = path.join(__dirname, 'data', 'Dockerfile')
-  const dockerfile = load(dockerfilePath)
-  expect(dockerfile).toBeInstanceOf(AlpineImage)
```

Comment:
I just realized that we don't have tests on package management anymore (expect for the unsupported case).
Ideally, we should cover package detection on apk and apt.